src/components/shared/Home/TagsListHoriz.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import FilterPill from "@/components/shared/FilterPill";
+
+interface TagsListHorizProps {
+  tagList: string[] | null;
+}
+
+const TagsListHoriz: React.FC<TagsListHorizProps> = ({ tagList }) => {
+  return (
+    <div className="flex flex-wrap gap-2">
+      {tagList?.map((tag) => {
+        return (
+          <FilterPill
+            icon={""}
+            key={tag}
+            text={tag}
+            backgroundColor="bg-myBlack-700"
+            textColor="text-myWhite-300"
+            filterType="tag"
+          />
```

Comment:
```suggestion
          <FilterPill
            key={tag}
            text={tag}
            backgroundColor="bg-myBlack-700"
            textColor="text-myWhite-300"
            filterType="tag"
          />
```
For props that we want as optional- let's just make them optional in the type for the component. We can avoid having to pass null data or empty strings by doing this to the component's types:

```js
// FilterPill.tsx
interface FilterPillProps {
  icon?: string; // the question mark here means we don't have to pass it in.
  text: string;
  backgroundColor: string;
  textColor: string;
  filterType: string;
}
```

Adding the question mark makes it 'undefined' so you don't need to pass empty strings in when using it.